runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/NamespaceRegistry.java

```diff
@@ -326,6 +327,14 @@ BindingRegistry findBinding(
         return bindingsById.get(bindingId);
     }
 
+    public KindConfig findBindingKind(
```

Comment:
Please rename to `resolveBindingKind`.

Reply:
renamed to `resolveKind` as per your previous comment